src/components/Modal/Modal.jsx

```diff
@@ -273,11 +285,18 @@ Modal.propTypes = {
    * is not the first focusable element in the document
    */
   initialFocusSelector: PropTypes.string,
+  /**
+   * Analytics tracking function(s) will be called
+   */
+  enableAnalytics: PropTypes.bool,
```

Comment:
I'd recommend reversing this. Call it `disableAnalytics`. That way it tracks by default, and if we really don't want it to, we tell it to not with `<Modal disableAnalitycs />` instead of `<Modal enableAnalytics={false} />`.